lib/semantic_logger/formatters/json_with_api_metadata.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module SemanticLogger
+  module Formatters
+    class JsonWithApiMetadata < Raw
+      def call(log, logger)
+        super(log, logger).merge(api_metrics).to_json
+      end
+
+    private
+
+      def api_metrics
+        return {} unless payload
+
+        {
+          params: payload[:params],
+          exception: payload[:exception],
+          current_user_class: payload[:current_user_class],
```

Comment:
bah just realised we add this already in the API controller: https://github.com/DFE-Digital/early-careers-framework/blob/main/app/controllers/api/api_controller.rb#L24

As I am seeing it twice in the responses, maybe we drop the current user class/id as its a duplicate and keep the other two?